website/src/css/custom.css

```diff
@@ -43,6 +43,12 @@
   --color-nav-text: #bfc0c0;
 }
 
+.dropdown--version--hide .dropdown__menu {
+  opacity: 0 !important;
```

Comment:
Is it necessary to use the !important flags here? I generally like to stay away from them if possible. But it's also possible it's entirely necessary in this situation. What do you think?

Reply:
Looks like it still works without the `!important` flag, removed in latest commit